src/main/java/DAO/CustomerDAO.java

```diff
@@ -0,0 +1,58 @@
+package DAO;
+
+import Models.*;
+import Utils.HibernateSessionFactoryUtil;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+
+import java.util.List;
+
+public class CustomerDAO {
+
+    public CustomerDAO() {
+
+    }
+
+    public Customer findById(int id) throws Exception {
+        return HibernateSessionFactoryUtil.getSessionFactory().openSession().get(Customer.class, id);
+
+    }
+
+    public void saveCustomer(Customer cust) {
```

Comment:
could saveCustomer() return some value to ensure that the operation finished.

Reply:
It's print: Hibernate: insert into Customer (age, e_mail, first_name, last_name, phone) values (?, ?, ?, ?, ?)
Do you want to return some number value?